src/main/resources/templates/Index.html

```diff
@@ -1,85 +1,87 @@
     <button class="open-button" onclick="openForm()">Log In</button>
-
-
-    <p th:text="${volunteer}">
-
-
+    <p th:text="${volunteer}"></p>
 
     <div class="form-popup" id="myForm1">
-      <form action="#" th:action="@{/saveVolunteer}" th:object="${volunteer}"method="GET" class="form-container">
+      <form action="#" th:action="@{/saveVolunteer}" th:object="${volunteer}" method="GET" class="form-container">
 
-        <label for="email"><b>Email</b></label>
-        <input type="text" placeholder="Enter Email" name="email" th:field="*{volunteerEmail}" required>
+        <label for="email1"><b>Email</b></label>
+        <input type="text" placeholder="Enter Email" name="email1" id="email1" th:field="*{volunteerEmail}" required>
 
-        <label for="psw"><b>Password</b></label>
-        <input type="password" placeholder="Enter Password" name="psw" th:field="*{volunteerPass}" required>
+        <label for="psw1"><b>Password</b></label>
+        <input type="password" placeholder="Enter Password" name="psw1" id="psw1" th:field="*{volunteerPass}" required>
 
         <button type="submit" class="btn">Login</button>
-        <button type="button" class="btn cancel" onclick="closeForm()">Close</button>
+        <button type="button" class="btn cancel" onclick="closeForm1()">Close</button>
       </form>
     </div>
 
     <button class="open-button" onclick="openForm()">Sign Up</button>
 
-    <div class="form-popup" id="myForm">
-      <form action="/action_page.php" class="form-container">
+    <div class="form-popup" id="myForm2">
+      <form action="/action_page.php" class="form-container" th:object="${volunteer}">
+
+        <label for="email3"><b>Email</b></label>
+        <input type="text" placeholder="Enter Email" name="email3" id="email3" th:field="*{volunteerEmail}" required>
 
+        <label for="email4"><b>Username</b></label>
+        <input type="text" placeholder="Enter Email" name="email4" id="email4" th:field="*{volunteerUser}" required>
 
-        <label for="email"><b>Email</b></label>
-        <input type="text" placeholder="Enter Email" name="email" th:field="*{volunteerEmail}" required>
-        <label for="email"><b>Username</b></label>
-        <input type="text" placeholder="Enter Email" name="email" th:field="*{volunteerUser}" required>
+        <label for="psw2"><b>Password</b></label>
+        <input type="password" placeholder="Enter Password" name="psw2" id="psw2" th:field="*{volunteerPass}" required>
 
-        <label for="psw"><b>Password</b></label>
-        <input type="password" placeholder="Enter Password" name="psw1" th:field="*{volunteerPass}" required>
-        <label for="psw"><b>Re-enter Password</b></label>
-        <input type="password" placeholder="Enter Password" name="psw2" th:field="*{volunteerPass}" required>
+        <label for="psw3"><b>Re-enter Password</b></label>
+        <input type="password" placeholder="Enter Password" name="psw3" id="psw3" th:field="*{volunteerPass}" required>
```

Comment:
Properly defined `for` attribute for `<label>` by defining `id` attribute for `input` tags being labeled. [Reference](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label)